### PR TITLE
Package structure

### PR DESCRIPTION
**Pull request by brgcode**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -72,43 +89,90 @@
...
+        attr = getattr(obj, attr_name)
+        if inspect.isfunction(attr):
+            filename = inspect.getmodule(obj).__name__.replace('.', '/')
+            lineno = inspect.getsourcelines(attr)[1]
+        else:
+            return None
+    else:
+        return None
+
+    return f"https://github.com/dbt-ethz/compas_vol/blob/master/src/{filename}.py#L{lineno}"
```
Comment (worbit) [inline docs/conf.py:149]: should the branch `master` in this url also be renamed to `main`?

Comment (worbit) [reply docs/conf.py:149]: thanks for adding and updating all the files. looks good to me from what I see

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
```
Comment (gonzalocasas) [inline docs/conf.py:149]:
```suggestion
    return f"https://github.com/dbt-ethz/compas_vol/blob/main/src/{filename}.py#L{lineno}"
```

File: requirements-dev.txt [hunk 3]
```diff
@@ -1,11 +1,19 @@
-sphinx>=1.6
-sphinx_compas_theme>=0.3
-invoke>=0.14
-bumpversion>=0.5
-check-manifest>=0.36
-pytest>=3.2
-twine
+autopep8
+attrs >=17.4
+bump2version >=0.5.11
```
Comment (gonzalocasas) [inline requirements-dev.txt:3]:
```suggestion
bump2version >=1.0

File: setup.cfg [hunk 4]
```diff
@@ -25,4 +29,8 @@ addopts =
 
 [isort]
 force_single_line = True
-line_length = 180
\ No newline at end of file
+line_length = 180
+known_first_party = compas
```
Comment (gonzalocasas) [inline setup.cfg:33]:
```suggestion
known_first_party = compas_vol
```

File: setup.cfg [hunk 5]
```diff
@@ -25,4 +29,8 @@ addopts =
...
+default_section = THIRDPARTY
+forced_separate = test_compas
```
Comment (gonzalocasas) [inline setup.cfg:35]:
```suggestion
forced_separate = test_compas_vol
```

File: tasks.py [hunk 6]
```diff
@@ -95,49 +90,71 @@ def clean(ctx, docs=True, bytecode=True, builds=True):
 
         if builds:
             folders.append('build/')
-            folders.append('src/compas_vol.egg-info/')
+            folders.append('src/compas.egg-info/')
```
Comment (gonzalocasas) [inline tasks.py:93]:
```suggestion
            folders.append('src/compas_vol.egg-info/')
```

File: tasks.py [hunk 7]
```diff
@@ -170,6 +187,27 @@ def prepare_changelog(ctx):
...
 
 
+@task(help={
+      'gh_io_folder': 'Folder where GH_IO.dll is located. Defaults to the Rhino 6.0 installation folder (platform-specific).',
+      'ironpython': 'Command for running the IronPython executable. Defaults to `ipy`.'})
+def build_ghuser_components(ctx, gh_io_folder=None, ironpython=None):
+    """Build Grasshopper user objects from source"""
+    with chdir(BASE_FOLDER):
+        with tempfile.TemporaryDirectory('actions.ghcomponentizer') as action_dir:
+            target_dir = source_dir = os.path.abspath('src/compas_ghpython/components')
```
Comment (gonzalocasas) [inline tasks.py:197]:
```suggestion
            target_dir = source_dir = os.path.abspath('src/compas_vol/ghpython/components')
```

File: .github/pull_request_template.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+- [ ] Other (e.g. doc update, configuration, etc)
+
+### Checklist
+
+_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
+
+- [ ] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
+- [ ] I ran all tests on my computer and it's all green (i.e. `invoke test`).
+- [ ] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
+- [ ] I added new functions/classes and made them available on a second-level import, e.g. `compas.datastructures.Mesh`.
```
Comment (gonzalocasas) [inline .github/pull_request_template.md:19]:
```suggestion
- [ ] I added new functions/classes and made them available on a second-level import, e.g. `compas_vol.primitives.VolBox`.
```

**Review by gonzalocasas — APPROVED** [review]
Added a couple of tiny suggestions here and there but 👍

**Pull request merged**